charts/argo-cd/templates/argocd-server/role.yaml

```diff
@@ -42,3 +42,11 @@ rules:
   verbs:
   - create
   - list
+{{- if eq (toString (index (coalesce .Values.server.config .Values.configs.cm) "exec.enabled")) "true" }}
```

Comment:
Do we need an AND condition here to the namespaced / server.clusterAdminAccess option @jmeridth @pdrastil?

Reply:
@mkilchhofer I don't think we need any additional condition for this PR.

The question is if we actually need these checks for Role and ClusterRole as the Argo UI also requires proper permission in Argo RBAC or AppProject. Without these permissions terminal is not shown in the UI.

Reply:
Agree. Let's merge it :)